### PR TITLE
RSWEB-7802: Remove 300 font weight, and increase line-height/base font height.

### PR DESCRIPTION
**Pull request by amberRrucker**
style(fonts):removing 300 font weight, increasing line height and base font height.

Can be previewed across the styleguide. Should check here:
http://localhost:9000/derek/solutions/solutions 
and here:
http://localhost:9000/derek/assets/typography

### CODE REVIEW

File: styleguide/_themes/derek/scss/patterns/cards/keyOffers.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@
 }
 
 .keyoffersWidget-body {
-  line-height: 1.7em;
+  //line-height: 1.7em;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/patterns/cards/keyOffers.scss]: Should we just remove this line instead?

File: styleguide/_layouts/default.jade [hunk 2]
```diff
@@ -16,7 +16,7 @@ html(lang="en")
 
     link(href="#{ css }", rel="stylesheet")
     link(href="css/main.css", rel="stylesheet")
-    link(href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600", rel="stylesheet")
+    link(href="https://fonts.googleapis.com/css?family=Open+Sans:400,600", rel="stylesheet")
```
Comment (WidgetsBurritos) [inline styleguide/_layouts/default.jade]: Do you know, is there a www ticket to remove the font weight there as well?

**Comment by amberRrucker** [comment]
@WidgetsBurritos you're super right and I just removed that line.

**Comment by WidgetsBurritos** [comment]
👍 from a code perspective.

**Comment by WidgetsBurritos** [comment]
Running through this with EJ we found a few issues:

The rug is a bit awkward now:
screen shot 2016-12-08 at 2 35 06 pm

The nav has a weird bottom border:
screen shot 2016-12-08 at 2 35 23 pm

Also it appears lead content (underneath the title) are still getting a font-weight: 300 from somewhere. Not entirely sure where it's coming in, but we're still seeing it in there.

**Comment by amberRrucker** [comment]
I've fixed the issues with the navigation and the rug. There is a quick and dirty fix for the rug that I'm not sure will fly with everyone but the alternatives are .... annoying when we're taking out the rug soon anyways?
@WidgetsBurritos @ejchiefs

**Comment by bighappyface** [comment]
I am still getting the bottom padding in the main navigation.

screen shot 2016-12-09 at 10 06 17 am

**Comment by amberRrucker** [comment]
@bighappyface, 
I don't think I was looking at it quite wide enough. Should be good now.

File: styleguide/_themes/global/scss/bootstrap_overrides.scss [hunk 3]
```diff
@@ -47,6 +47,15 @@
   }
 }
 
+//Quick and Dirty fix before we take out the rug.
+.fa-3x {
+  font-size: 2.5em!important;
```
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/bootstrap_overrides.scss]: If we're gonna do something hacky, I'd rather do something like this instead:
```
.rug-container .fa-3x { font-size: 2.5em; }
```

Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/bootstrap_overrides.scss]: If we're gonna do something hacky, I'd rather do something like this instead:

```
.rug-container .fa-3x { font-size: 2.5em; }
```

**Comment by amberRrucker** [comment]
@WidgetsBurritos got it! thanks.

**Comment by bighappyface** [comment]
Desktop looks good.

The nav is still a bit off for mobile:

screen shot 2016-12-09 at 12 45 48 pm
screen shot 2016-12-09 at 12 45 30 pm

**Comment by amberRrucker** [comment]
@bighappyface alrighty! got it. Thank you.

**Comment by runrog** [comment]
👍 much smoother

**Pull request merged**